### PR TITLE
Feature/discrim writer 136

### PR DESCRIPTION
**Pull request by guiguem**
Not working yet: it compiles but the Test fails...
I am not sure why yet!

### CODE REVIEW

**Comment by guiguem** [comment]
The strange part here is that if I replace the TDiscrimatedPoint by Cicada::TProcessedTrack (in the TSparseWaterfallCandidateData::TSparseWaterfallCandidateData() and KT2ROOT::LoadSparseWaterfallCandidateData(...,...), it just works.
This is sooo strange!

**Comment by guiguem** [comment]
@cclaessens @nsoblath I think it works: this PR should solve #136

File: Source/IO/Conversions/KTROOTData.cc [hunk 1]
```diff
@@ -18,486 +18,562 @@
...
+    //     AllTrackData track( dummyData, procTrack );
+
+    //     for (Int_t iTrack = 0; iTrack < nTracks; ++iTrack)
+    //     {
+    //         ((TProcessedTrackData*)((*fTracks)[iTrack]))->Unload(procTrack);
+    //         ((TClassifierResultsData*)((*fClassifierResults)[iTrack]))->Unload(classData);
+    //         data.AddTrack(track);
+    //     }
+    //     return;
+    // }
```
Comment (cclaessens) [inline Source/IO/Conversions/KTROOTData.cc]: why is all this commented? will it be removed? is it temporarily commented?

**Comment by guiguem** [comment]
This is an issue Evan is going to handle I think, right @evzayas ?

**Pull request merged**